src/js/components/location/LocationButton.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { RiMapPinLine } from 'react-icons/ri';
+
+const LocationButton = ({ location, onClick }) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const buttonStyle = () => {
+    if (!location.backgroundColor || ['FFFFFF', 'FFFF'].includes(location.backgroundColor)) {
```

Comment:
For locations inside a modal there is a feature that applies a color on **LocationButton** based on the `location.backgroundColor` value 
![image](https://user-images.githubusercontent.com/35233347/188607894-17d121e1-2777-4f83-af62-216f4934493c.png)

The problem was that by default most of the locations have a white color _(and not a consistent value like `#FFFFFF` and sometimes `#FFFF`)_